packages/nodes-base/nodes/Github/Github.node.ts

```diff
@@ -538,13 +538,15 @@ export class Github implements INodeType {
 								type: 'string',
 								default: '',
 								description: 'The name of the author of the commit.',
+								placeholder: 'i18n TODO DELETE Sample name',
```

Comment:
here are some TODOs in the file, leftovers?

Reply:
These were introduced on purpose and will be removed. i18n can only translate strings that exist, and I needed to test if a param with this type, level of nesting, etc. that had a particular property, so this was the fastest. All can be identified with `i18n TODO DELETE`